modules/mauth-signer-akka-http/src/test/scala/com/mdsol/mauth/http/ImplicitsTest.scala

```diff
@@ -135,20 +135,14 @@ class ImplicitsTest extends AnyWordSpec with Matchers {
           RawHeader(MAuthRequest.MCC_TIME_HEADER_NAME, "mcc-time-value"),
           RawHeader(MAuthRequest.MCC_AUTHENTICATION_HEADER_NAME, "mcc-authentication-value"),
           RawHeader(MAuthRequest.X_MWS_TIME_HEADER_NAME, "x-mws-time-value"),
-          RawHeader(MAuthRequest.X_MWS_AUTHENTICATION_HEADER_NAME, "x-mws-authentication-value"),
-          RawHeader("Content-Type", "application/json")
+          RawHeader(MAuthRequest.X_MWS_AUTHENTICATION_HEADER_NAME, "x-mws-authentication-value")
         ).toString()
```

Comment:
Can we also add a wiremock test to ensure that the request has the right content-type header?

Reply:
Added wiremock tests to check that the default content type is text/plain UTF-8 and also, that the httpclient is sending the customized content-type FYI @austek